### ISSUE TITLE
The direction is not sensitive

### ISSUE THREAD

**Issue by 2936175613** [CuntBoy/Snake#2]
I pressed up and left direction quickly when I was nearly died,but it may only response left direction.
I think  up has been covered by left,or the response time is too long for me.

**Comment by 2936175613** [comment]
And I suggest addding an operation for pausing

**Comment by CuntBoy** [comment]
you can press space to pause game

**Comment by CuntBoy** [comment]
> I pressed up and left direction quickly when I was nearly died,but it may only response left direction.
> I think up has been covered by left,or the response time is too long for me.

**Comment by CuntBoy** [comment]
> I pressed up and left direction quickly when I was nearly died,but it may only response left direction.
> I think up has been covered by left,or the response time is too long for me.
Indeed, it will only respond to your most recent press, at a fixed time. I set the response time to 200 milliseconds
